packages/shared/src/utils/date/__tests__/date-time.spec.ts

```diff
@@ -1,3 +1,4 @@
+/* eslint-disable no-unused-expressions */
```

Comment:
I think it's better to move this ignore line before the line that is ignored to know the exact line that has the error to be fixed later.

Reply:
I got this error in the whole file! I'm not sure why this happening so I ignored this rule in the whole app!

Reply:
I think this error happen on lines 69 and 75.

The reason for this error is this line:

```
import { expect } from 'chai';
```
I think we need to replace `chai` methods with `jest` and remove this line. We have eslint rules for ' jest` and `chai` are incompatible with `chai` methods.

Reply:
Replacing chai with jest is out of this card's scope! let's change them later in a separate PR!
but about eslint ignore I moved that to line 68!

Reply:
Yes, you're right, I just comment this for having this in mind and make plan for it. you can ignore it in this PR. thanks.